components/admin_console/billing/billing_history.tsx

```diff
@@ -186,43 +183,59 @@ const BillingHistory: React.FC<Props> = () => {
-                                year='numeric'
-                            />
-                        </td>
-                        <td>
-                            <div>{info.product_name}</div>
-                            <div className='BillingHistory__table-bottomDesc'>{info.charge_desc}</div>
-                        </td>
-                        <td className='BillingHistory__table-total'>
-                            <FormattedNumber
-                                value={info.total}
-                                // eslint-disable-next-line react/style-prop-object
-                                style='currency'
-                                currency='USD'
-                            />
-                        </td>
-                        <td>
-                            {getPaymentStatus(info.status)}
-                        </td>
-                        <td className='BillingHistory__table-invoice'>
-                            {info.invoice &&
-                                <a href={info.invoice}>
+                {billingHistory.map((invoice: Invoice) => {
+                    const fullUsers = invoice.line_items.filter((item) => item.metadata.type === 'full').reduce((val, item) => val + item.quantity, 0);
+                    const partialUsers = invoice.line_items.filter((item) => item.metadata.type === 'partial').reduce((val, item) => val + item.quantity, 0);
+
+                    return (
+                        <tr
+                            className='BillingHistory__table-row'
+                            key={invoice.id}
+                        >
+                            <td>
+                                <FormattedDate
+                                    value={new Date(invoice.period_start)}
+                                    month='2-digit'
+                                    day='2-digit'
+                                    year='numeric'
+                                />
+                            </td>
+                            <td>
+                                <div>{product?.name}</div>
+                                <div className='BillingHistory__table-bottomDesc'>
+                                    <FormattedMarkdownMessage
+                                        id='admin.billing.history.usersAndRates'
+                                        defaultMessage='{fullUsers} users at full rate, {partialUsers} users with partial charges'
+                                        values={{
+                                            fullUsers,
+                                            partialUsers,
+                                        }}
+                                    />
+                                </div>
+                            </td>
+                            <td className='BillingHistory__table-total'>
+                                <FormattedNumber
+                                    value={invoice.total}
```

Comment:
Invoice total comes as an int where the last two digits are the decimal so we need